config/sync/field.field.paragraph.banner.field_banner_title.yml

```diff
@@ -0,0 +1,24 @@
+uuid: b82247ae-c04f-4bac-bb68-64337c111264
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.paragraph.field_banner_title
+    - filter.format.underlined_title
+    - paragraphs.paragraphs_type.banner
+  module:
+    - text
+id: paragraph.banner.field_banner_title
+field_name: field_banner_title
+entity_type: paragraph
+bundle: banner
+label: 'Banner Title'
+description: ''
+required: true
+translatable: false
+default_value: {  }
+default_value_callback: ''
+settings:
+  allowed_formats:
+    - underlined_title
+field_type: text_long
```

Comment:
I'm unsure on some things for this field. 

1. I think you should decrease the amount from rows from 5 to for example 2. I don't think a title should ever be longer than that, and there's no need for that many empty lines. 

2. I understand that this is banner paragraph, but the implementation for the underline is something that i assume should be use/replaced across multiple paragraphs since this behavior is wanted multiple places. In that regard i would consider: 
- Renaming this from explicitly using "banner" in the machine name, and make it more generic. This would make more sense it if became re-used across more paragraphs.
- Creating a ticket describing a task to update other paragraphs with the new underline implementation and get DDF to check it / write which paragraphs should be updated-